docs/cli-kit/ui-kit/contributing.md

```diff
@@ -0,0 +1,241 @@
+# Contributing to UI Kit
+
+## What is UI Kit
+
+UI Kit is a library that can be used to either ask for input from the user or render output to the terminal.
+By using UI Kit you're making sure that whatever you're rendering in the terminal will follow our design system.
+This ensures familiar-feeling interaction patterns for the users and allows you to benefit from design improvements without
+having to change your code too often.
+
+At its core, UI Kit is built with [ink](https://github.com/vadimdemedes/ink) which uses React under the hood.
```

Comment:
Nitpick: `ink` ~> `Ink`

Reply:
Yeah I haven't been consistent with that, I'll change it to `Ink` everywhere.